package.json

```diff
@@ -1,17 +1,28 @@
 {
-  "name": "@locker/near-membrane",
-  "version": "0.4.0",
+  "name": "@salesforce/near-membrane",
+  "private": true,
+  "workspaces": [
+    "packages/*",
+    "examples"
+  ],
   "description": "JavaScript Near Membrane Library to create a sandboxed environment",
   "module": "lib/index.js",
   "types": "types/index.js",
   "scripts": {
-    "build": "yarn build:types && rollup -c && yarn build:examples",
-    "build:types": "tsc --declaration true --emitDeclarationOnly true --declarationDir types",
-    "build:examples": "rollup -i examples/web-components/platform.js -o examples/web-components/bundle.js -f cjs",
-    "start": "rollup -c --watch -w & http-server ./examples",
+    "clean": "lerna run clean --stream",
+    "build": "lerna run build --stream",
+    "start": "http-server ./examples",
     "test": "jest --runInBand --no-cache --config ./jest.config.js",
     "test:bundlesize": "bundlesize",
-    "publish": "npm publish --access public",
+    "version": "yarn build",
+    "bump": "lerna version --exact --no-push --yes",
+    "bump:patch": "lerna version patch --exact --no-push --yes",
+    "bump:minor": "lerna version minor --exact --no-push --yes",
+    "bump:beta": "lerna version prerelease --preid beta --exact --no-push --yes",
+    "bump:test": "lerna version prerelease --preid test --exact --no-push --yes",
+    "push": "git push && git push --tags",
```

Comment:
For now, we need a manual push from localhost until we have a CI job for publishing